test/backends/file_test.py

```diff
@@ -44,8 +44,9 @@
     ids=lambda x: str(x[0]),
 )
 def user_file(request):
-    with storage.Backend(*request.param) as backend:
-        yield backend
+    backend_file, can_detect_sector_size = request.param
+    with backend_file as backend:
+        yield storage.Backend(backend, can_detect_sector_size)
```

Comment:
We don't really need to get the result of the context manager, so we can do:

    backend, can_detect_sector_size = request.param
    with backend:
        yield storage.Backend(backend, can_detect_sector_size)